pkg/controller/node/nodecontroller.go

```diff
@@ -453,11 +507,18 @@ func (nc *NodeController) monitorNodeStatus() error {
 						nc.evictPods(node.Name)
 						continue
 					}
-
-					if err := nc.kubeClient.Nodes().Delete(node.Name); err != nil {
+					freeCIDR := ""
+					if node.Spec.PodCIDR != "" {
+						freeCIDR = node.Spec.PodCIDR
+					}
+					err = nc.kubeClient.Nodes().Delete(node.Name)
+					if err != nil {
```

Comment:
Please revert the split - we're using the form `if err := ...; err != nil {` everywhere.